apps/login-flow/src/app/app.tsx

```diff
@@ -1,14 +1,36 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import { Admin, Login } from '@form-exercise/core/web';
+import { Admin, Login, NotFound } from '@form-exercise/core/web';
+import i18n from 'i18next';
+import { I18nextProvider } from 'react-i18next';
+import { MenuApp } from '@form-exercise/core/web';
+import ErrorBoundary from 'libs/core/web/src/lib/error-handler/error-boundary/ErrorBoundary';
+import { initI18Next } from '@form-exercise/i18n';
+import { ThemeContext } from '@form-exercise/ui';
+
+initI18Next();
 
 export function App() {
   return (
-    <BrowserRouter>
-      <Routes>
-        <Route path="/" element={<Login />} />
-        <Route path="/admin" element={<Admin />} />
-      </Routes>
-    </BrowserRouter>
+    <I18nextProvider i18n={i18n}>
+      <BrowserRouter>
+        <ThemeContext>
+          <Routes>
+            <Route element={<MenuApp />}>
+              <Route path="/" element={<Login />} />
```

Comment:
create a route configuration that contains all references to route paths, then use it to set the path here and in other places when using react-router api